app/src/components/SupportedLanguagesMenu/supportedTranslationLanguages.js

```diff
@@ -0,0 +1,148 @@
+export const SUPPORTED_TRANSLATION_CODES = [
```

Comment:
NIT: Should we rename this to ordered_supported_translation_codes? and add a comment of the reason we decided to choose this ordering rather than just iterating over LANGUAGE_NAMES?

Reply:
I can do that but iterating over language names was never an option as objects don't have stable key orders

Reply:
Wouldn't Maps help solve the problem?

Reply:
Yes but I tend to generally not use them due to their mutability. My goal is to move to https://immutable-js.github.io/immutable-js/ over time and use `OrderedMap` from it here